### DISCUSSION TITLE
Creating Tar File

### DISCUSSION THREAD

**Discussion by jimph** [micropython/micropython#10456] [General]
Hi,

utarfile has support for extracting from a tar file (X.tar -> files), but I do not see any support for creating a tar file (files -> X.tar). 
Can someone point me to any existing library/ code snippet for creating a new tar?

Thanks,
Jim

**Comment by jimmo** [comment]
@jimph I'm not aware of any existing library to do this.

If you were interested in writing this yourself, two things that might be useful:
 - The CPython implementation of tarfile -- https://github.com/python/cpython/blob/3.11/Lib/tarfile.py
 - MicroPython's existing (extract-only)utarfile implementation -- https://github.com/micropython/micropython-lib/blob/master/micropython/utarfile/utarfile.py
